### PR TITLE
Implements Mocha and Repository

### PR DESCRIPTION
**Pull request by DuardoN**

### CODE REVIEW

**Comment by mateusbs17** [comment]
@DuardoN I suggest you finish this PR with [PR/12](https://github.com/OpenTesseract/aletheia/pull/12) changes, it has travis.yml implemented.

File: packages/server/api/controller/personalityController.js [hunk 1]
```diff
@@ -1,50 +1,64 @@
 'use strict';
 
-const mongoose = require('mongoose');
-
-const Personality = mongoose.model('Personality');
-// Personality.relationship({ path: 'speechs', ref: 'Speech', refPath: 'personality' });
+const PersonalityRepository = require('../repository/personality');
 
+// I think controllers files can be a class and not exporting functions
```
Comment (mateusbs17) [inline packages/server/api/controller/personalityController.js]: Go for it, I would like to update the syntax to node10 as soon as possible.

File: packages/server/routes/personality.js [hunk 2]
```diff
@@ -10,27 +11,67 @@ const router = require('../lib/util').router();
 /**
  * GET {domain}/personality
  */
-router.get('/', personality.listAll);
+router.get('/', listAll);
```
Comment (mateusbs17) [inline packages/server/routes/personality.js]: no need to create the function listAll, just pass it as a callback. Also, use arrow functions.

**Comment by thesocialdev** [comment]
@DuardoN the travis failure is a linting error
```
$ eslint --cache --max-warnings 0 --ext .js --ext .json .
/home/travis/build/OpenTesseract/aletheia/packages/server/api/repository/personality.js
  22:18  error  Parsing error: Unexpected token update
✖ 1 problem (1 error, 0 warnings)
```

File: packages/server/api/repository/personality.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Personality = require('../model/personalityModel');
+
+/**
+ * @class PersonalityRepository
+ */
+module.exports = class PersonalityRepository {
+    static listAll() {
```
Comment (thesocialdev) [inline packages/server/api/repository/personality.js]: use arrow functions

**Pull request merged**